### PR TITLE
Fix some flake8 errors

### PR DESCRIPTION
**Pull request by tehw0lf**
Hello there, I have taken a look at some of the formatting issues :)

Several files have been flake8ified, but I left the star imports as they are. They should be reviewed to reach full flake8 consistency in the future.

### CODE REVIEW

File: subjects/traversal.py [hunk 1]
```diff
@@ -2,45 +2,49 @@ class TraversalUtils:
...
-        #SELECT name from widget_subject where lft <= (left traversal) and rght >= (right traversal) order by lft ASC;
-        #Grabs all parent objects from the browse_subject all the way up to the root
-        parent_subjects = current_object.objects.filter(lft__lte=browse_subject.lft, rght__gte=browse_subject.rght).order_by('lft')
+        # This statment is equivalent to:
+        # SELECT name from widget_subject where lft <= (left traversal) and
+        # rght >= (right traversal) order by lft ASC;
+        # Grabs all parent objects from the browse_subject all the way up to the root
+        parent_subjects = current_object.objects.filter(lft__lte=browse_subject.lft,
+                                                        rght__gte=browse_subject.rght
+                                                        ).order_by('lft')
```
Comment (somexpert) [inline subjects/traversal.py]: So I think personally I like the look of 
```
parent_subjects = current_object.objects.filter(
    lft__lte=browse_subject.lft,
    right__gte=browse_subject.right
).order_by('lft')
```
Mind making it look like that?

File: subjects/traversal.py [hunk 2]
```diff
@@ -50,26 +54,30 @@ def get_structured_list(self, current_object, browse_subject):
...
             if subject_id != 0:
-                #Grabs the row object of the given node
+                # Grabs the row object of the given node
                 current_subject = subject.objects.get(id__exact=subject_id)
-            #Grabs the children objects of the given node
-            self.children_subjects = subject.objects.filter(parent__exact=subject_id).order_by('name')
+            # Grabs the children objects of the given node
+            self.children_subjects = subject.objects.filter(
+                                                            parent__exact=subject_id
+                                                            ).order_by('name')
```
Comment (somexpert) [inline subjects/traversal.py]: I'm also not a fan of the way this was split. How about:
```
self.children_subjects = subject.objects.filter(
    parent__exact=subject_id
).order_by('name')
```

File: subjects/traversal.py [hunk 3]
```diff
@@ -50,26 +54,30 @@ def get_structured_list(self, current_object, browse_subject):
...
-            #Grabs the row object of the given node
+            # Grabs the row object of the given node
             current_subject = subject.objects.get(name__exact=subject_name)
             subject_id = current_subject.id
-            #Grabs the children objects of the given node
-            self.children_subjects = subject.objects.filter(parent__exact=subject_id).order_by('name')
+            # Grabs the children objects of the given node
+            self.children_subjects = subject.objects.filter(
+                                                            parent__exact=subject_id
+                                                            ).order_by('name')
```
Comment (somexpert) [inline subjects/traversal.py]: Same as above.

Comment (tehw0lf) [reply subjects/traversal.py]: You are totally right about that, it does not look clean.

**Comment by somexpert** [comment]
Thank you for the beautification, @tehw0lf! :thumbsup:

**Pull request merged**